dask/array/fft.py

```diff
@@ -10,6 +11,9 @@
 except ImportError:
     scipy = None
 
+from dask.array.creation import linspace as _linspace
+from dask.array.core import concatenate as _concatenate
```

Comment:
Sorry, was trying to ask a different question, which maybe is being answered. Would it better to use `import dask.array as da` and just use `da` for these or is it better to import these specific functions and use them instead?